components/molecule/photoUploader/src/DragState.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import AtomIcon, {ATOM_ICON_SIZES} from '@s-ui/react-atom-icon'
+
+import {BASE_CLASS_NAME} from './config'
+const DRAG_STATE_CLASS_NAME = `${BASE_CLASS_NAME}-dragState`
+const DEFATUL_DRAG_STATE_STATUS = 'accepted'
+
+const DragState = ({icon, status = DEFATUL_DRAG_STATE_STATUS, text}) => {
+  const dropzoneClassName = cx(DRAG_STATE_CLASS_NAME, {
+    [`${DRAG_STATE_CLASS_NAME}--accepted`]: status === 'accepted',
+    [`${DRAG_STATE_CLASS_NAME}--rejected`]: status === 'rejected'
+  })
+
+  return (
+    <div className={dropzoneClassName}>
+      <AtomIcon size={ATOM_ICON_SIZES.extraLarge}>{icon()}</AtomIcon>
```

Comment:
Why don't we pass the component already instanced? One level yup we already know which one we have to pass right? So we could instance it as well! Maybe I'm missing something, I'm interested why you did it this way